style.css

```
@@ -0,0 +1,367 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+header{
+    text-align: center;
+}
+
+.section{
```

always have space before {
```suggestion
.section {
```